lib/plugins/config/admin.php

```diff
@@ -77,24 +77,26 @@ public function html()
         $allow_debug = $GLOBALS['conf']['allowdebug']; // avoid global $conf; here.
         global $lang;
         global $ID;
+		
+		define('kHeadlineLevel', '2'); /* change this if doc structure requires */
 
         $this->setupLocale(true);
 
         echo $this->locale_xhtml('intro');
 
-        echo '<div id="config__manager">';
+        echo '<article id="config__manager">';
 
         if ($this->configuration->isLocked()) {
-            echo '<div class="info">' . $this->getLang('locked') . '</div>';
+            echo '<p class="highlight">' . $this->getLang('locked') . '</p>';
```

Comment:
This changes the severity of the shown message and should be reverted.

Reply:
Changed back to "info" and added an "info"-icon to it (was apparently missing in the file... it is now inlined in the CSS).